app/assets/javascripts/modules/gtm-reorder-listener.js

```diff
@@ -0,0 +1,20 @@
+var GtmReorderListener = {}
+
+GtmReorderListener.handleReorder = function (eventName, dataGtm) {
+  return function (event) {
+    var element = event.target
+
+    if (!element || !eventName || !dataGtm) {
```

Comment:
Minor: we don't really need this, since the first one doesn't get used anywhere, and the latter two we set ourselves in the same file.

Reply:
Good call – I removed them.